src/ossia-max/src/ossia-max.cpp

```diff
@@ -532,12 +481,32 @@ std::vector<std::string> parse_tags_symbol(t_symbol** tags_symbol, long size)
   return tags;
 }
 
-void ossia_max::discover_network_devices(ossia_max* x)
+void ossia_max::discover_network_devices(ossia_max*)
 {
   ossia_max::zeroconf_oscq_listener.browse();
   ossia_max::zeroconf_minuit_listener.browse();
   clock_delay(ossia_max::browse_clock, 100.);
 }
 
+void ossia_max::push_parameter_value(ossia::net::parameter_base* param, const ossia::value& val)
+{
+  if(!param_locks.contains(param))
+  {
+    param_locks.push_back(param);
+    param->push_value(val);
+    param_locks.remove_all(param);
+  }
+}
+
+void ossia_max::set_parameter_value(ossia::net::parameter_base* param, const ossia::value& val)
+{
+  if(!param_locks.contains(param))
+  {
+    param_locks.push_back(param);
+    param->set_value(val);
```

Comment:
this is wrong, `ossia::net::parameter::set_value()` doesn't do what we need
in Max the set message should set the internal value without firing it through object's outlet. 
Thus in ossia-max the set message should set the value and propagate it to other objects and network without firing it.